app/src/main/res/values/donottranslate.xml

```diff
@@ -16,7 +14,6 @@
     <string name="submission_properties_seperator_comments">"•"</string>
 
     <string name="submission_properties_seperator">"  •  "</string>
-    <string name="general_pro">Slide for Reddit Pro</string>
```

Comment:
I didn't even press "delete" on any resources. The linter just did it because you didn't use these strings. Up to you if you want them to stick around.